[bad-underlying-sinks.dedicatedworker.html]
  [start: errors in start cause WritableStream constructor to throw]
    expected: FAIL

  [close: throwing method should cause writer close() and ready to reject]
    expected: FAIL

  [close: returning a rejected promise should cause writer close() and ready to reject]
    expected: FAIL

  [close: throwing getter should cause writer close() to reject]
    expected: FAIL

  [write: throwing getter should cause write() and closed to reject]
    expected: FAIL

  [write: throwing method should cause write() and closed to reject]
    expected: FAIL

  [write: returning a promise that becomes rejected after the writer write() should cause writer write() and ready to reject]
    expected: FAIL

  [write: returning a rejected promise (second write) should cause writer write() and ready to reject]
    expected: FAIL

  [abort: non-function abort method with .apply]
    expected: FAIL

  [abort: throwing getter should cause abort() and closed to reject]
    expected: FAIL

  [abort: throwing method should cause abort() and closed to reject]
    expected: FAIL

  [close: throwing getter should cause constructor to throw]
    expected: FAIL

